creator-cards/ricardo-perez.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ricardo Josue Perez Altamirano",
+  "type": "CREATOR",
+  "tags": ".NET, C#, ASP.NET, I LOVE DOTNET,Azure",
+  "github": "https://github.com/RicardoJosuePerezAltamirano",
+  "channels": [
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/c/RicardoJosue"
+    },{
+      "name": "Sessionize",
+      "url": "https://sessionize.com/ricardo-josue-perez-altamirano/"
+    },{
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/ricardojosue/"
```

Comment:
@DeeJayTC Can LinkedIn and YouTube be added here as well, or should it be only in the sign-up form?

Reply:
I set this info in the sign up form, and I set here for better contact but if not it is the place for this data i will remove

Reply:
Sorry, this got lost as i apparently accidently resolved the comment, we don't need all this as channels as the content on the page would be heavily duplicated. You can add social infos via the page itself, can you remove the channels here and we merge your feed?

Reply:
Ready in commit 7fc2d21933509928a5d67fdd14433949083dcc37